npm/qsharp/generate_katas_content.js

```diff
@@ -20,7 +20,9 @@ import { basename, dirname, join, relative, sep } from "node:path";
 import { fileURLToPath } from "node:url";
 
 import mdit from "markdown-it";
-const md = mdit("commonmark").disable(["escape"]);
```

Comment:
Note how previously I had to disable the default escape processing in Markdown when converting. No longer needed!